.doc_gen/metadata/cognito-identity-provide_metadata.yaml

```diff
@@ -0,0 +1,17 @@
+# zexi 0.4.0
+cognito-identity-provider_ListUserPools:
+  title: List the &COGIDP; associated with an &AWS; account using an &AWS; SDK
+  title_abbrev: List the user pools
+  synopsis: list the &COGIDP;.
```

Comment:
Same as title, so:
`list &COGUP; for an account.`

Reply:
Fixed